var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -68,7 +68,7 @@ class Cp2k(MakefilePackage, CudaPackage):
     variant('lmax',
             description='Maximum supported angular momentum (HFX and others)',
             default='5',
-            values=list(HFX_LMAX_RANGE),
+            values=['{0}'.format(i) for i in HFX_LMAX_RANGE],
```

Comment:
```suggestion
            values=map(str, HFX_LMAX_RANGE),
```
This is a bit simpler

Reply:
Thank you! Always nice to learn something.
Fabien